### PR TITLE
feat(eslint): re-enable max-length rule and fix errors

### PR DESCRIPTION
**Pull request by undrivendev**
# Description
re-enable max-length ESLint rule and fix errors. Related to #803.

### CODE REVIEW

File: src/app/features/issue/issue.service.ts [hunk 1]
```diff
@@ -120,7 +120,9 @@ export class IssueService {
     }
 
     for (const issuesType of Object.keys(tasksIssueIdsByIssueType)) {
-      const updates = await (this.ISSUE_SERVICE_MAP[issuesType].refreshIssues as any)(tasksIssueIdsByIssueType[issuesType], isNotifySuccess, isNotifyNoUpdateRequired);
+      const updates = await (this.ISSUE_SERVICE_MAP[issuesType].refreshIssues as any)(tasksIssueIdsByIssueType[issuesType],
```
Comment (johannesjo) [inline src/app/features/issue/issue.service.ts:123]: This is a bit ugly I'd say. maybe we can set the max line length to a more convenient value as a quick fix? Alternatively I much prefer this:
```TypeScript
  const updates = await (this.ISSUE_SERVICE_MAP[issuesType].refreshIssues as any)( 
        tasksIssueIdsByIssueType[issuesType],
        isNotifySuccess,
        isNotifyNoUpdateRequired
  );
```

File: src/app/features/tag/store/tag.effects.ts [hunk 2]
```diff
@@ -237,7 +237,8 @@ export class TagEffects {
     tap((arg) => console.log('Error INFO Today:', arg)),
     tap(({activeId, allTasks}) => {
       const allIds = allTasks.map(t => t && t.id);
-      const r = confirm('Nooo! We found some tasks with no data. It is strongly recommended to delete them to avoid further data corruption. Delete them now?');
+      const r = confirm('Nooo! We found some tasks with no data.\
```
Comment (johannesjo) [inline src/app/features/tag/store/tag.effects.ts:240]: I don't think that max line length adds much value here. I would ignore this line.

File: src/app/imex/sync/check-for-update.util.ts [hunk 3]
```diff
@@ -38,7 +38,9 @@ export const checkForUpdate = (params: { remote: number, local: number, lastSync
     } else if (lastSync < local) {
       return UpdateCheckResult.RemoteUpdateRequired;
     } else if (lastSync === local) {
-      alert('Sync Warning: Dropbox date not up to date despite seemingly successful sync. (This might happen when: 1. You have conflict changes and decide to take the local version. 2. You open the other instance and also decide to use the local version.)');
+      alert('Sync Warning: Dropbox date not up to date despite seemingly successful sync.\
```
Comment (johannesjo) [inline src/app/imex/sync/check-for-update.util.ts:41]: I also would rather ignore this line.

File: src/app/util/model-version.ts [hunk 4]
```diff
@@ -10,7 +10,8 @@ export const isMigrateModel = (modelData: any, localVersion: number, modelType:
   } else if (importVersion > localVersion) {
     const isNewMajor = ((Math.floor(importVersion) - Math.floor(localVersion)) >= 1);
     if (isNewMajor) {
-      alert('Cannot load model. Version to load is newer than local. Please close the app and update your local productivity version first, before importing the data.');
+      alert('Cannot load model. Version to load is newer than local. Please close\
```
Comment (johannesjo) [inline src/app/util/model-version.ts:13]: Same thing.

**Review by johannesjo — CHANGES_REQUESTED** [review]

File: src/app/features/tag/store/tag.effects.ts (hunk 2 above)
Comment (undrivendev) [reply src/app/features/tag/store/tag.effects.ts:240]: What about disabling the max-length rule only for strings? There is a configuration option available for that: https://eslint.org/docs/rules/max-len#ignorestrings

Comment (undrivendev) [reply src/app/features/tag/store/tag.effects.ts]: Should be ok now, I disabled the max-length for strings

File: src/app/features/issue/issue.service.ts (hunk 1 above)
Comment (undrivendev) [reply src/app/features/issue/issue.service.ts]: Should be ok now

File: src/app/imex/sync/check-for-update.util.ts (hunk 3 above)
Comment (undrivendev) [reply src/app/imex/sync/check-for-update.util.ts]: Should be ok now, I disabled the max-length for strings

File: src/app/util/model-version.ts (hunk 4 above)
Comment (undrivendev) [reply src/app/util/model-version.ts]: Should be ok now, I disabled the max-length for strings

**Pull request merged**